Source/WebCore/Modules/applepay/cocoa/PaymentAPIVersionCocoa.mm

```diff
@@ -34,8 +34,11 @@
 
 unsigned PaymentAPIVersion::current()
 {
-    static unsigned current = [] {
-#if ENABLE(APPLE_PAY_FEATURES)
+    static const unsigned current = [] {
+#if ENABLE(APPLE_PAY_DEFERRED_PAYMENTS)
+        // FIXME: No public version 15 yet, so should Deferred Payments be in 15 like Features?
+        return 16;
```

Comment:
The version number should be frozen.

See https://github.com/WebKit/WebKit/pull/2793 for more details.

Reply:
Oh I see, excellent!
Should I add a comment there, so that others won't be tempted to change it again?

Reply:
I've added a comment, but let me know if you'd prefer I didn't touch this file.

Reply:
A comment is a good idea!

I would say: 

> This version number should not be changed anymore, features can now be found in method.data.features.

Rather than:

> This version number should not need to change anymore, features may now be found in method.data.features.